mu-plugins/blocks/global-header-footer/postcss/_common.pcss

```diff
@@ -31,6 +31,30 @@ html {
 	}
 }
 
+/**
+ * Set up custom properties.
```

Comment:
I think it'd be good to document that these are duplicates (both here and at the original source), so that folks remember to update them in both places when they change in the future.